src/platform/Linux/NetworkCommissioningWiFiDriver.cpp

```diff
@@ -214,6 +283,77 @@ bool LinuxWiFiDriver::WiFiNetworkIterator::Next(Network & item)
     return true;
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFI_PDC
+CHIP_ERROR LinuxWiFiDriver::AddOrUpdateNetworkWithPDC(ByteSpan ssid, ByteSpan networkIdentity,
+                                                      Optional<uint8_t> clientIdentityNetworkIndex, Status & outStatus,
```

Comment:
Should consider passing the ByteSpans and Optional by const reference, not by value.

Reply:
The other methods in this particular class all pass ByteSpans by value, so I followed that pattern. E.g.
```
Status AddOrUpdateNetwork(ByteSpan ssid, ByteSpan credentials, MutableCharSpan & outDebugText, uint8_t & outNetworkIndex)
```